apps/passport-server/test/genericIssuance.spec.ts

```diff
@@ -746,10 +807,223 @@ t2,i1`,
+      expectLength(semaphoreGroupAttendees.value.members, 3);
+      expect(semaphoreGroupAttendees.value.members).to.deep.include.members([
+        EdgeCityDenverAttendeeIdentity.commitment.toString(),
+        EdgeCityManualAttendeeIdentity.commitment.toString(),
+        // Bouncer2 has a specially configured "superuser email", but is not
+        // a holder of a bouncer-tier ticket. Having a "superuser email" allows
+        // a user to perform check-ins, but does not change the product type of
+        // their ticket, and so does not change their Semaphore group
+        // memberships.
+        EdgeCityBouncer2Identity.commitment.toString()
+      ]);
+
+      const semaphoreGroupAttendeesAndBouncers =
+        await requestGenericIssuanceSemaphoreGroup(
+          process.env.PASSPORT_SERVER_URL as string,
+          edgeCityDenverPipeline.id,
+          edgeCitySemaphoreGroupIds.attendeesAndBouncers
+        );
+
+      expectTrue(semaphoreGroupAttendeesAndBouncers.success);
+      expectLength(semaphoreGroupAttendeesAndBouncers.value.members, 5);
+      expect(
+        semaphoreGroupAttendeesAndBouncers.value.members
+      ).to.deep.include.members([
+        EdgeCityBouncerIdentity.commitment.toString(),
+        EdgeCityBouncer2Identity.commitment.toString(),
+        EdgeCityDenverAttendeeIdentity.commitment.toString(),
+        EdgeCityManualAttendeeIdentity.commitment.toString(),
+        EdgeCityManualBouncerIdentity.commitment.toString()
+      ]);
+    }
+  );
+
+  step(
+    "New users can sign up, get added to group, prove group membership",
+    async function () {
+      expectToExist(giService);
+      const pipelines = await giService.getAllPipelines();
+      expectToExist(pipelines);
+      expectLength(pipelines, 3);
+      const edgeCityDenverPipeline = pipelines.find(LemonadePipeline.is);
+      expectToExist(edgeCityDenverPipeline);
+
+      // Test that a new user is added to the attendee group
+
+      const newUser = lemonadeBackend.addUser(
+        "newuser@example.com",
+        "New",
+        "User"
+      );
+      const newUserIdentity = new Identity();
+      EdgeCityLemonadeAccount.addUserTicket(
+        EdgeCityDenver._id,
+        EdgeCityAttendeeTicketType._id,
+        newUser._id,
+        newUser.name
+      );
+      await edgeCityDenverPipeline.load();
+      const edgeCityDenverTicketFeedUrl =
+        edgeCityDenverPipeline.issuanceCapability.feedUrl;
+      // The pipeline doesn't know that the user exists until they hit the feed
+      const NewUserTickets = await requestTicketsFromPipeline(
+        edgeCityDenverPipeline.issuanceCapability.options.feedFolder,
+        edgeCityDenverTicketFeedUrl,
+        edgeCityDenverPipeline.issuanceCapability.options.feedId,
+        ZUPASS_EDDSA_PRIVATE_KEY,
+        newUser.email,
+        newUserIdentity
+      );
+      expectLength(NewUserTickets, 1);
+
+      {
+        // Necessary to trigger Semaphore group update
```

Comment:
I could make some kind of public method for reloading the Semaphore groups, but I'm not sure I like making public methods just so that we can call them from tests.